data/human/free worlds side plots.txt

```diff
@@ -11,6 +11,85 @@
 # You should have received a copy of the GNU General Public License along with
 # this program. If not, see <https://www.gnu.org/licenses/>.
 
+
+
+mission "FW Convoy to Zug"
+	name "Convoy to Zug"
+	description "Escort these Haulers to Zug for an important project. Expect pirate raids."
+	source
+		government "Free Worlds"
+		near Zubeneschamali 2 4
+	destination "Zug"
+	to offer
+		has "FW Katya 7C: done"
```

Comment:
```suggestion
		has "event: war begins"
		"combat rating" > 200
		or
			not "chosen sides"
			event "joined the free worlds"
```

Instead of restricting this based on the player formally being involved in the FW story, how about basing it on the start of the war?

If I did my suggestion correctly, this would make the player able to get this mission if one of two circumstances are true:

A: The war has started, the player has a combat rating over 200, and they have not committed to choosing sides yet.
B: The war has started, the player has a combat rating over 200, and they have formally joined the Free Worlds.

There is a small window in between the two when this mission wouldn't be available when the player is technically working for the FW council but not actually formally a member, but it's relatively small. On the plus side, this does open this mission up to being done by independent captains that happen to be flying around in that part of space, which seems reasonable.